### ISSUE TITLE
Did I brick my Coral Dev Board?

### ISSUE THREAD

**Issue by JasonCruz-Dev** [google-coral/edgetpu#540]
### Description

Hi,

Thanks ahead of time for reading this.

So my friend that has zero knowledge plugged in my Google Coral Dev Board without my permission or knowledge. Once I figured out what he was up too, I got onto him about it. And I guess from the whole situtation, he booted it up the board without the sd and fireware, when I got onto him about messing with my things, he immediately pulled out the power. So thats error #1. Booting up the drive without an SD/fireware installed. After I started reading the docs, I flashed the SD card, inserted it, and booted up the Dev Board, about half way the power at the office went out. Go figure.. So I didnt have any other way of making sure this would work, because of 2 errors in already. Regardless not knowing, I turned it on again once the power got back on, and let it go through its process, the little red light stayed on for about 5mins, then turned off on its own. At this point im thinking everything looks to be working. I removed the SD card, changed the DIP switch 1,0,0,0, and restarted it. At this point  I already had all the software installed and its been nothing but trial and error to get it working. 

I can not get it up and running for the life of me. 

I used a virtual machine inside Windows 10, ran Ubuntu 20 version, installed python and pip, and also installed MDT. Ive tried going throught the root, and not in the root. Idk what I could be doing wrong, or is it bricked because of the first 2 major erros from the beginning? When I pull up ~$ mdt devices or mdt wait-for-device, and I either get nothing or I get another error saving I have too many process going.. Ill take a screen shot if you need one. Other than all that I said, what can you tell me I got wrong, do I have to buy another dev board?

Any help would be apprecatied! 
Thanks
J

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug, Build/Install, Others

### Operating System

Mendel Linux, Linux, Ubuntu

### Coral Device

Dev Board

### Programming Language

Python 3.9, Other

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
Hello @JasonCruz-Dev  can you try to connect with [mdt](https://coral.ai/docs/dev-board/get-started/#requirements) or [serial console](https://coral.ai/docs/dev-board/serial-console/) on native windows 10 instead of virtual machine. Thanks!

**Comment by JasonCruz-Dev** [comment]
hi thanks for reaching out. If this what im doing nojw doesnt work, thats my next thought. I tried the fastboot like everyone suggested on other threads but not success. ill try windows 10 and see what i can come up with.

**Comment by JasonCruz-Dev** [comment]
Hi

So I got further going through windows 10. Installing all the packages was a challenge. But the most difficult challenge that I had/have is using powershell to run `mdt wait-for-device` or `mdt reboot-bootloader` or even `mdt shell` but with non of those commands was I able to see the dev board boot into mendel linux fireware.

I did however, notice my windows 10 laptop recognize the USB's once I connected them to the laptop. When I checked the Device manager to see if it gets recognized there, I got drivers that had errors, so with alittle bit of research, I found some USB drivers that would work. But now that I''ve update the drivers for the USB COM ports, in Terminal i'm getting 2 errors. See picture below. 

I figured if my laptop recognizes the USB COM ports, let me see what module or driver I'll need to connect powershell with the dev boards. But Im stuck here for now, until I can figure out how to fix this step. Let me know your thoughts. Thanks!

![Screenshot 2022-02-18 232322](https://user-images.githubusercontent.com/54033391/154791527-92e61083-14a1-4568-9799-5638359f4b9d.png)

**Comment by JasonCruz-Dev** [comment]
Also, now I use to get 2 different blinking lights on the dev board, now the only orange lights blinks ( see picture) . I wish there was documentation that could tell us what 31 flashes of amber says what the problem is for.
![20220219_013145](https://user-images.githubusercontent.com/54033391/154791669-96fb7f47-5a81-4f5f-b00d-84b786a37b67.jpg)

**Comment by JasonCruz-Dev** [comment]
ok. So for that problem. I had both cables plugged into USB 2.0 Ports and not USB 3.0 Ports. I changed that and now im not getting any errors. But still no `mdt devices` will show for me. Or `mdt shell`. This is where I am at.

![Screenshot 2022-02-18 235756](https://user-images.githubusercontent.com/54033391/154792400-fb502852-1d4c-40bc-a852-ee16baea2f8e.png)

**Comment by JasonCruz-Dev** [comment]
after looking at google documentation, there isnt any. 

Only for linux and macOS

this is troubling. And the MDT service is from what I can see only for when you want to connect through ethernet. But in order to do that, you must first connect through the USB first, BEFORE using the MDT service or connecting through it via ethernet.

This is what I have so far. I did swap out monitors and switched to a TV instead for picture. And I did notice something boot, but what? Now all I have is a screen that looks like theres water in the background. So maybe that is it? At this point I have no ideea. I just want to connect from my terminal to my DevBoard. (frustrated)

**Comment by JasonCruz-Dev** [comment]
I would like to thank myself for getting through this. 

If anybody needs help, connect with me i got you. Ill help you figure it out. I made it. lots of trial and errors. I used Putty on Windows and this [Documentation](https://coral.ai/docs/dev-board/serial-console/#connect-with-windows) once you have installed everything, python, pip, flashed your microsd card, and even done your flashed bios, login in using Putty for Windows 10. Do not use Powershell or Git trying to connect using `mdt wait-for-device` because that is for connecting to the device over internet.

Thats my experience.
![Screenshot 2022-02-19 214416](https://user-images.githubusercontent.com/54033391/154830348-f674b823-367d-4bad-9752-e43cdb406246.png)

`mdt` was good for helping me detect devices, but it only detected a device once I plugged in my ethernet cable. It showed me the device name for the DevBoard, but not serial port. 

Anyone reading this, make sure to install your device drivers, luckily I found some drivers for the PC.
